pkg/scaffold/gopkgtoml.go

```diff
@@ -92,6 +92,10 @@ required = [
   branch = "master" #osdk_branch_annotation
   # version = "=v0.3.0" #osdk_version_annotation
 
+[[override]]
+   name = "k8s.io/kube-openapi"
```

Comment:
Can we move this up with the other k8s.io overrides?

Also, looks like an extra space made it in:
```suggestion
  name = "k8s.io/kube-openapi"
```

Reply:
@LiliC We still need to change this right?